### PR TITLE
Zen tidy up

### PR DESCRIPTION
**Pull request by richard-boeve**

### CODE REVIEW

**Comment by ben-vandelaar** [comment]
Hey has this been checked yet?

**Review by EmilSchroder — COMMENTED** [review]
A few things I would consider based on the formatting guide https://www.granneman.com/webdev/coding/formatting-and-indenting-your-html/:

I think inline elements like <a> and <abbr> should be kept as part of the paragraph flow instead of separated onto their own line. This helps readability.

I would use the <p></p> like you have one lines 31 & 34 in the other instances, expect independent, without them sharing the line with other text/tags.

With the list <li> tags, keeping the <li></li> on the same line would also improve readability.

**Comment by EmilSchroder** [comment]
It looks like github comments has actually tried to use the tags I was referencing. Let me try again:

A few things I would consider based on the formatting guide https://www.granneman.com/webdev/coding/formatting-and-indenting-your-html/:

I think inline elements like <*a*> and <*abbr*> and should be kept as part of the paragraph flow instead of separated onto their own line. This helps readability.

I would use the <*p*><*/p*> like you have one lines 31 & 34 in the other instances, expect independent, without them sharing the line with other text/tags.

With the list <*li*> tags, keeping the <*li*><*/li*> on the same line would also improve readability.

File: pristine-zen-garden.html [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+    <title>CSS Zen Garden: The Beauty of CSS Design</title>
+
+    <link rel="stylesheet" media="screen" href="/214/214.css?v=8may2013">
+    <link rel="alternate" type="application/rss+xml" title="RSS" href="http://www.csszengarden.com/zengarden.xml">
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Dave Shea">
+    <meta name="description" content="A demonstration of what can be accomplished visually through CSS-based design.">
+    <meta name="robots" content="all">
+
```
Comment (richard-boeve) [inline pristine-zen-garden.html]: test

**Comment by richard-boeve** [comment]
Thanks for the comments, good change suggestions. I've applied your suggested changes. If all good, please go ahead and merge and remove the 'tidying' branch.

**Comment by EmilSchroder** [comment]
I don't have the option to merge the pull request. Can you double check I'm added as a collaborator?

**Comment by richard-boeve** [comment]
Yep you are. Reading the ticket again, it says "Tidy their repository by deleting the tidying branch (they'll still have a copy locally if they need it)."
So maybe I am jumping the gun by assuming it needed to be merged? Might be sprint 3 :-)

**Review by EmilSchroder — APPROVED** [review]

**Comment by EmilSchroder** [comment]
The guy who was doing mine managed to merge and delete the tidying branch, so I honestly don't know. I looked up some videos and the only thing that seems to make sense is that I somehow don't have "write access", however being a collaborator should have given me that. 

I am stumped.
